server/src/main/java/org/elasticsearch/index/engine/LiveVersionMap.java

```diff
@@ -448,20 +448,20 @@ synchronized void clear() {
 
     @Override
     public long ramBytesUsed() {
-        return maps.current.ramBytesUsed.get() + ramBytesUsedTombstones.get();
+        return maps.current.ramBytesUsed.get() + maps.old.ramBytesUsed.get() + ramBytesUsedTombstones.get();
```

Comment:
can we maybe add `Maps#ramBytesUsed()` it would prevent this issue if its used again somewhere?